<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NiosSpiCmdDialog</class>
 <widget class="QDialog" name="NiosSpiCmdDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send NIOS/SPI command</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="lineEdit_Data">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DataRead">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Command">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Data">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Status">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_ReadData">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Read data from NIOS</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_CardIndex">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_CardIndex">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Card index</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Timeout">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_Timeout">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Timeout (ms)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_DataRead">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data from NIOS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Cmd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Use &quot;0x&quot; prefix for hexadecimal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_DataRead_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0x</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Send</sender>
   <signal>clicked()</signal>
   <receiver>NiosSpiCmdDialog</receiver>
   <slot>s_SendCmd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>s_SendCmd()</slot>
 </slots>
</ui>
